docs/faq.md

```diff
@@ -0,0 +1,82 @@
+
+## Is there a central list of all hooks and tasks available in Tool Kit?
+
+Funny you should ask! We have just the thing:
+
+| Hook                | Hook exported by plugin | Possible tasks       |
+|---------------------|-------------------------|----------------------|
+| `run:local`         | npm                     | `Node`               |
+|                     |                         | `Nodemon`            |
+|                     |                         | `NextRouter`         |
+|                     |                         | `WebpackDevelopment` |
+|                     |                         | `WebpackWatch`       |
+| `test:local`        | npm                     | `Mocha`              |
+|                     |                         | `JestLocal`          |
+|                     |                         | `Eslint`             |
+|                     |                         | `SecretSquirrel`     |
+|                     |                         | `CypressLocal`       |
+| `build:local`       | npm                     | `BabelDevelopment`   |
+|                     |                         | `WebpackDevelopment` |
+| `build:ci`          | circleci                | `BabelProduction`    |
+|                     |                         | `WebpackProduction`  |
+| `test:ci`           | circleci                | `Eslint`             |
+|                     |                         | `Mocha`              |
+|                     |                         | `JestCi`             |
+|                     |                         | `CypressCi`          |
+| `deploy:review`     | circleci-heroku         | `HerokuReview`       |
+| `deploy:staging`    | circleci-heroku         | `HerokuStaging`      |
+| `test:review`       | circleci-heroku         | `NTest`              |
+|                     |                         | `Pa11y`              |
+| `test:staging`      | circleci-heroku         | `NTest`              |
+| `teardown:staging`  | circleci-heroku         | `HerokuTeardown`     |
+| `deploy:production` | circleci-heroku         | `HerokuProduction`   |
+| `publish:tag`       | circleci-npm            | `NpmPublish`         |
+| `cleanup:remote`    | heroku                  | `NpmPrune`           |
+| `release:remote`    | heroku                  | `UploadAssetsToS3`   |
+| `build:remote`      | heroku                  | `BabelProduction`    |
+|                     |                         | `WebpackProduction`  |
+| `git:precommit`     | husky-npm               | `LintStaged`         |
+|                     |                         | `SecretSquirrel`     |
+| `git:commitmsg`     | husky-npm               |                      |
+| `test:staged`       | lint-staged-npm         | `Eslint`             |
+| `format:staged`     | lint-staged-npm         | `Prettier`           |
+| `format:local`      | prettier                | `Prettier`           |
+
+[table updated December 2022]
```

Comment:
**thought**: should we update it for January 2023 with the TypeScript tasks? Also imo we should include a day of the month here too as we're still making quite a lot of changes each month!

Reply:
Good thought! Will do!